### PR TITLE
Register any modification after selection change

### PR DESCRIPTION
**Pull request by mightyiam**

### CODE REVIEW

**Comment by mightyiam** [comment]
Works lovely in all browsers.

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
   "devDependencies": {
     "lodash.times": "^3.0.1",
     "combokeys": "^2.4.3",
-    "edited": "^1.1.1",
+    "edited": "^1.1.2",
```
Comment (mightyiam) [inline package.json]: I love fixing things upstream.

Comment (winhamwr) [inline package.json]: Minor quibble, but I would say that this is more of a new feature in `edited` than a bugfix, so I probably would have bumped it to `1.2.0`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,8 @@ WYMeditor.
 * [#764](https://github.com/wymeditor/wymeditor/pull/764)
   TablePlugin row/column remove:
   refrain from error when no selectedContainer
+* [#749](Register any modification after selection change)
```
Comment (winhamwr) [inline CHANGELOG.md]: How about:

If a user modifies their selection via arrow keys or clicking, a new undo/redo savepoint is now automatically created. Previously, typing, moving selection, typing some more and then clicking undo would remove both typing actions. Now, only the second bit of typing will be undone.

File: package.json (hunk 1 above)
Comment (mightyiam) [inline package.json]: In the scope of `edited`, it does seem like a feature, now that you say it.

File: CHANGELOG.md (hunk 2 above)
Comment (mightyiam) [inline CHANGELOG.md]: I made it better.

**Comment by mightyiam** [comment]
Thank you!

**Pull request merged**